horizons/constants.py

```diff
@@ -635,7 +635,7 @@ class NETWORK:
 	# change port to 2022 for development server updated after UH commits
 	SERVER_PORT = 2002
 	CLIENT_ADDRESS = None # type: Optional[str]
-	UPDATE_FILE_URL = "http://updates.unknown-horizons.org/current_version.php"
+	UPDATE_FILE_URL = "http://unknown-horizons.org/current-version.json"
```

Comment:
At least not directly with GitHub. I wonder if we could setup something before it, Cloudflare maybe? Then we could use `https://unknown-horizons.github.io`